### PR TITLE
Automapper

### PR DESCRIPTION
**Pull request by ShreenidhiHD**
updated mapper and Unit test

### CODE REVIEW

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Controllers/SampleController.cs [hunk 1]
```diff
@@ -4,28 +4,30 @@
 using Microsoft.Extensions.Logging;
 using TestingAndCalibrationLabs.Business.Core.Interfaces;
 using Microsoft.AspNetCore.Hosting;
-
+using AutoMapper;
+using TestingAndCalibrationLabs.Business.Core;
 namespace TestingAndCalibrationLabs.Web.UI.Controllers
 {
     /// <summary>
     /// a base class for view
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Controllers/SampleController.cs:12]: what is this comment, not able to make sense of it.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs/SampleTest.cs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+using TestingAndCalibrationLabs.Business.Core.Interfaces;
+using TestingAndCalibrationLabs.Business.Core.Model;
+using TestingAndCalibrationLabs.Business.Data.Repository.Interfaces;
+using TestingAndCalibrationLabs.Business.Services;
+using TestingAndCalibrationLabs.Web.UI.Controllers;
+
+namespace TestingAndCalibrationLabs.Tests
+{
+    /// <summary>
+    /// Decaring Class
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs/SampleTest.cs:17]: Please give proper comments, not making sense.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs/SampleTest.cs [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+            // resolve dependencies.
+            _mapper = mapper.Object;
+            _sampleRepository = sampleRepo.Object;
+            _sampleService = new SampleService(_sampleRepository);
+        }
+
+        /// <summary>
+        /// Creating atest For getting all Details from Database
+        /// </summary>
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs/SampleTest.cs:71]: Comments should explain what is test scenario we are covering.

**Review by smartdusttechnologies — COMMENTED** [review]
1. do we need to check0in below files?
TestingAndCalibrationLabs/.vs/TestingAndCalibrationLabs/v17/.suo
TestingAndCalibrationLabs/.vs/TestingAndCalibrationLabs/v17/.futdcache.v1
TestingAndCalibrationLabs/.vs/TestingAndCalibrationLabs/DesignTimeBuild/.dtbcache.v2
TestingAndCalibrationLabs/TestingAndCalibrationLabs.DB/TestingAndCalibrationLabs.DB.sqlproj.user
2. Please give comment on the methods of
TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Data/Repository/Interfaces/Sample/ISampleRepository.cs

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs/SampleTest.cs (hunk 2 above)
Comment (ShreenidhiHD) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs/SampleTest.cs:17]: Added proper comments

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Controllers/SampleController.cs (hunk 1 above)
Comment (ShreenidhiHD) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Controllers/SampleController.cs:12]: changed comments

**Review by smartdusttechnologies — APPROVED** [review]
ok

**Pull request merged**